services/JSONFileService.js

```diff
@@ -0,0 +1,30 @@
+const fs = require("fs").promises;
+
+class JSONFileService {
+  constructor(filePath) {
+    this.filePath = filePath;
+  }
+
+  getData() {
+    return fs
+      .readFile(this.filePath, "utf8")
+      .then((data) => JSON.parse(data))
+      .catch((error) => {
+        throw error;
+      });
+  }
+
+  updateData(newData) {
+    const data = JSON.stringify(newData, null, 2);
+    return fs
+      .writeFile(this.filePath, data, "utf8")
+      .then(() => {
+        console.log("Data updated successfully.");
+      })
+      .catch((error) => {
+        throw error;
+      });
+  }
+}
+
+module.exports = JSONFileService;
```

Comment:
export an instance
```suggestion
module.exports = new JSONFileService();
```

Reply:
The constructor in JSONFileService accepts file path of our db. It's better to keep it this way for future extensions of our app, because this service might be used in other places in the future.

Reply:
add fallback path and if we do not pass any custom in props - use fallback